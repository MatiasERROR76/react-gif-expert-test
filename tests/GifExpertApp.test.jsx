import { useState } from "react";
import { fireEvent, render, renderHook, screen } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe("Pruebas en <GifExpertApp/>", () => {
  const inputValue = "Saitama";

  test("should match the snapshot", () => {
    const { container } = render(<GifExpertApp />);
    expect(container).toMatchSnapshot();
    // screen.debug();
  });

  test("Should return the initial state", () => {
    const { result } = renderHook(() => useState([]));
    const [categories] = result.current;
    expect(categories.length).toBe(0);
  });
});
