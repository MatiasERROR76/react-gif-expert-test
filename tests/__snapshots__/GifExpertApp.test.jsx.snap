// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp/> should match the snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="buscar gifs"
      type="text"
      value=""
    />
  </form>
  <h2>
    One Punch
  </h2>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
